### PR TITLE
updated mui deps and added not found message

### PR DESCRIPTION
**Pull request by abelpz**
#### Describe what your pull request addresses (Please include relevant issue numbers):
- Updated MUI material dependencies to solve tree view results not showing with newer versions due to https://mui.com/blog/lab-tree-view-to-mui-x/

- Fixed playground not working because of missing theme-provider.

#### Please include detailed Test instructions for your pull request:
- Run the playground by executing the command `pnpm nx serve fnr-pg` (to install dependencies run `pnpm install`), search one of the cards, results should show if any, a "No results found" message should show if none. 

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Is the repo documentation accurate/appropriate?
  - [ ] Check Stylguidist if applicable
  - [ ] Check readme for correct information about the feature being worked on
- [ ] Check that there are not inadvertent commits 
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story

### CODE REVIEW

File: packages/fnr-mui/src/results-tree/results-tree.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import { TreeView, TreeViewProps } from '@mui/lab';
+import { TreeView, TreeViewProps } from '@mui/x-tree-view';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import { devNull } from 'os';
```
Comment (kintsoogi) [inline packages/fnr-mui/src/results-tree/results-tree.tsx]: What is the `devNull` imported here being used for?

File: packages/fnr-mui/src/findr-results/findr-results.tsx [hunk 2]
```diff
@@ -59,7 +59,9 @@ export function FindrResults({
           }
         >
           {groupsEntries.map(([sourceKey, group], groupIndex) =>
-            !group?.results?.length ? null : (
+            !group?.results?.length
+              ? <Alert severity="info">No results found</Alert>
```
Comment (kintsoogi) [inline packages/fnr-mui/src/findr-results/findr-results.tsx:63]: The "no results found" is a great addition!

**Review by kintsoogi — CHANGES_REQUESTED** [review]
The update works and looks good to me!
I just left one question about an added import that seems to be unused

File: packages/fnr-mui/src/results-tree/results-tree.tsx (hunk 1 above)
Comment (abelpz) [reply packages/fnr-mui/src/results-tree/results-tree.tsx]: Some vscode extension added that automatically it must have suggested it to me and I accepted it by mistake.

**Review by kintsoogi — APPROVED** [review]
LGTM

**Pull request merged**